tools/algolia/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "algolia",
```

Comment:
Only just noticed this is in a separate package. Would much prefer if we can keep this all to a single `package.json` to minimise developer overhead. I'm not seeing what would be gained by having a separate package here.

Reply:
Fine by me.

My reason for doing this was that the algolia module is only required for doing Algolia indexing (not for running the app itself) and therefore made more sense in a separate package.json.